MarsSpecFlowProject/MarsSpecFlowProject/StepDefinitions/ThisTestSuiteContainsTestScenariosForLanguageTab_StepDefinitions.cs

```diff
@@ -19,7 +21,8 @@ public class ThisTestSuiteContainsTestScenariosForLanguageTab_StepDefinitions :
 
         Login loginobj = new Login();
 
-
+    LanguageWorkFlow  langobj = new LanguageWorkFlow();
+        LanguageAssertionHelpers assertobj = new LanguageAssertionHelpers();  
 
         [BeforeScenario]
         public void Setup()
```

Comment:
your hooks only need to be implemented once! please create a separate class and move the hooks there!

Reply:
https://codebun.com/how-to-implement-hooks-in-specflow-c/